woocommerce/woocommerce-android
# Add the missing store creation analytics

This PR adds tracking for store creation.

**Testing instructions**
Check the following in the logcat:

When clicking on "Get started" from prologue, creating an account, leaving the store creation: Tracked: `SITE_CREATION_DISMISSED`, Properties: `source` = `PROLOGUE`

When clicking on "Get started" from prologue, creating an account, finishing the store creation: Tracked: ``LOGIN_WOOCOMMERCE_SITE_CREATED``, Properties: `source` = `PROLOGUE`, `url` = [the new URL]

When clicking on "Log in" from sign up screen, then tapping on Create store button in the site picker: Tracked: `SITE_PICKER_CREATE_SITE_TAPPED`, Properties: `source` = `LOGIN`

When logged in, going to settings and then tapping on switch site, tapping on Create store button in the site picker: Tracked: `SITE_PICKER_CREATE_SITE_TAPPED`, Properties: `source` = `SWITCHING_STORE`
